### ISSUE TITLE
Accessibility - Text not read by screen reading software

### ISSUE THREAD

**Issue by timo54** [rokwire/illinois-app#19]
### Android 2.2.11

**Describe the bug**
The text shown in the red box on the screen capture below is not announced for screen readers. They will not know it's there. 

**To Reproduce**
1. Activate screen reading software - Talkback
2. Swipe until you get to the Events screen
3. Swipe down to an event and listen to hear if the Available times are read
4. Talkback does not announce the available times

**Expected behavior**
Each element on the screen should be focusable when swiping through the page so that screen reading software will announce it. 

**Screenshots**
See below

**Desktop (please complete the following information):**
 - OS: Android
- Version 2.2.11

**Smartphone (please complete the following information):**
 - Device: Google Pixel 3
 - OS: Android
 - Version 2.2.11

<img width="477" alt="NotReadText" src="https://user-images.githubusercontent.com/12497650/91504751-9c662d80-e893-11ea-9529-5bbb757f93e0.png">

**Comment by TodorBachvarov** [comment]
Fixed

**Comment by acovher2** [comment]
This bug is recurring in version 2.5.9 of app and I retested 7/12/2021 on iPhoneXR and Samsung galaxy s9.

**Comment by acovher2** [comment]
This is still present in 2.6.28.

**Comment by acovher2** [comment]
I'm unable to close this as it was opened by Tim
